usbsdmux/usbsdmux.py

```diff
@@ -122,6 +122,14 @@ def gpio_set_low(self, gpio):
         """
         raise NotImplementedError()
 
+    def info(self):
+        scr = self._usb.read_scr()
+        print("SCR: {}".format(scr.hex()))
+        cid = self._usb.read_cid()
+        print("CID: {}".format(cid.hex()))
+        csd = self._usb.read_csd()
+        print("CSD: {}".format(csd.hex()))
```

Comment:
I am fine with not adding any parsing in the initial PR adding this feature, but we will likely want to add parsing later on. We should not have to change the CLI interface in an backward-incompatible way once we do. Many users use the `usbsdmux` tool in scripts that should be mostly version agnostic.

How about moving the current `info` output behind a required `info --raw` flag and not supporting any other flags for now? This way we can add human readable parsed output later on using the bare `info` and `info --human` flags and machine readable ones using `info --json`.
I would even prefer having `info --json` as the first one we implement instead of `info --raw`, because we would extend it with parsed values later on, e.g.:

```bash
$ usbsdmux /dev/usb-sd-mux/id-00035.00002 info --json
{
  "SCR": {
    "raw": "0205808301000000"
  },
  "CID": {
    "raw": "9f544953444348456196e80009014a00"
  },
  "CSD": {
    "raw": "400e00325b590001cfff7f800a400000"
  }
}
```

Reply:
I intended the current one to be extended to a parsed output. Even with otherwise human readable output, you still want to have the raw strings for use with other decoders.

Reply:
My fault. It is even more complex:

* `usbmuxctl <command>` without any flag plots a ASCII-art representation of the USB-Mux state. (So this is the default.)
* `usbmuxctl --raw <command>` writes the status as `<key>=<value>` to stdout, so one can `grep` for the desired value.
* `usbmuxctl --json <command>` writes the status as `json`
* `usbmuxctl --quiet <command>` omits all output to `stdout`, but still sets the exit code accordingly.

Reply:
OK, so I'd leave the human readable format as the default. I'll add the json output.